torch/autograd/profiler_util.py

```diff
@@ -993,6 +1049,16 @@ def auto_scale_flops(flops):
         else:
             with_flops = False  # can't find any valid flops
 
+    if show_full_name:
+        if has_overload_names:
+            func_full_overload_name = "Full Overload Name"
+            headers.append(func_full_overload_name)
+            add_column(len(func_full_overload_name))
+
+        func_full_name = "Full Name"
+        headers.append(func_full_name)
+        add_column(len(func_full_name))
```

Comment:
please add comments here to notice the full name must be the last column of the table.

Reply:
Yes, updated.